README.md

```diff
@@ -37,7 +37,8 @@ Other useful links:
 - [Examples](https://examples.flame-engine.org/) of most features which can be tried out from your
   browser.
   - To access the code for each example, press the `< >` button in the top right corner.
-- [Tutorials](https://tutorials.flame-engine.org/) - A simple tutorial to get started.
+- [Tutorials](https://docs.flame-engine.org/1.6.0/tutorials/tutorials.html) - Some simple tutorials
```

Comment:
why link specifically to a version? that means we will have to keep updating this file every release

Reply:
Good catch, we really have to introduce `/latest`